### PR TITLE
[Merged by Bors] - Implement pause/stop

### PR DESCRIPTION
**Pull request by Maleware**
# Description

test https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/druid-operator-it-custom/102/

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Comment by Maleware** [comment]
Still WIP !

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +31,7 @@ All notable changes to this project will be documented in this file.
 [#407]: https://github.com/stackabletech/druid-operator/pull/407
 [#408]: https://github.com/stackabletech/druid-operator/pull/408
 [#415]: https://github.com/stackabletech/druid-operator/pull/415
+[#422]: https://github.com/stackabletech/hbase-operator/pull/336
```
Comment (maltesander) [inline CHANGELOG.md:34]:
```suggestion
[#422]: https://github.com/stackabletech/hbase-operator/pull/442
```

**Review by maltesander — COMMENTED** [review]
operator-rs update missing in "Changed" changelog entry

**Review by maltesander — APPROVED** [review]
bors merge

**Pull request closed without merging**